README.md

```diff
@@ -199,6 +199,13 @@ in the *AWS Command Line Interface User Guide*.
 The credentials must have a policy applied that
 [allows access to Amazon ECR](http://docs.aws.amazon.com/AmazonECR/latest/userguide/ecr_managed_policies.html).
 
+### Amazon ECR Docker Credential Helper 
+
+| Environment Variable  | Default Value | Description                                                       |
```

Comment:
The default values are empty for the first one and OS-dependent for the second, so it's a bit hard to describe them as "Default Value" here.

```suggestion
| Environment Variable  | Sample Value | Description                                                       |
```

Reply:
Chagned and rebased

https://github.com/awslabs/amazon-ecr-credential-helper/blob/efd1603705e91c31bf28cbbc5dba8c3e09c63100/ecr-login/config/cache_dir.go I thought though that we always return `"~/.ecr"` regardless of OS?  It would be nice to document the default value if possible but it's not a big deal if not

Reply:
My apologies; I thought we had already merged https://github.com/awslabs/amazon-ecr-credential-helper/pull/250 which changes the behavior.

Reply:
Ah, OK cool, I'll leave it as it is